tests/test_api/test_items.py

```diff
@@ -17,7 +17,6 @@ def test_get_item_present(self, test_client, item):
 
         data = json.loads(r.data)
         assert data['item']['title'] == item.title
-        assert data['item']['vote'] == 0
```

Comment:
Added back. I removed it when I was initially playing with splitting up item_fields before I realized that checking current_user.is_authenticated was the efficiency solution.